### ISSUE TITLE
The controlled vocabulary page needs to be more accessible when navigating via keyboard and using a screen reader.

### ISSUE THREAD

**Issue by Andrea-Guevara** [DSpace/dspace-angular#3366]
## Describe the bug
The controlled vocabulary page https://demo.dspace.org/browse/srsc is not very accessible to users who navigate DSpace via keyboard and users who make use of assistive technology. At first, it can be observed that when searching through the search bar, and no item is found, this is not announced to the user who uses a screen reader. Next, when using the “reset” button, the focus stays on the button and does not return to the search bar. And finally, there are several buttons without the area-label attribute.

## To Reproduce
1. Go to the controlled vocabulary page https://demo.dspace.org/browse/srsc
2. Navigate the page via keyboard using a screen reader

## Expected behavior
1. If the user using assistive technology makes a search on this page, and it gives no return, they need to be warned.
2. It would be more intuitive for the user to click on the “reset” button to return to the search bar. Giving them the idea that they can do a new search.
3. The presence of the aria-label attribute on buttons is important.
